SwiftWisdom/Core/StandardLibrary/Numbers/UnsignedInteger+Extensions.swift

```diff
@@ -8,38 +8,42 @@
 
 import Foundation
 
+// REVIEW: can we test this? how? 
 extension UnsignedInteger {
     public static func ip_random() -> Self {
         let intMax = Int(ip_maxValue.toIntMax())
-        let rand = randomInRange(0...intMax)
+        let rand = random(inRange: 0...intMax)
         return self.init(ip_safely: rand)
     }
 }
 
-public func randomInRange(_ range: ClosedRange<Int>) -> Int {
-    // TODO: Not good API design - since closed range does not assume positive only. We should either
-    // a) rename method to indicate positive only
-    // b) remove lower bound restriction
-    assert(range.lowerBound >= 0)
+// REVIEW: should this be moved into Int+Extension because it uses Int ?
```

Comment:
lets change this to "TODO:evaluate whether we should move random into CountableClosedRange<Int>"